STYLE.md

```diff
@@ -42,6 +42,18 @@ expression_that_prints_something
 # Line 3 of output
 ```
 
+## Filenames
+
+When indicating code that is written in a `.py` file, the following lines should appear at the top of the code block; the separating `-`s should reach exactly to the end of the `.py` extension in the filename, as below.
+```python
+# /usr/bin/env python3
+
+'''
+File: filename.py
+-----------------
+'''

```

Comment:
Can we remove the line between the shebang and use double quotes for the comment?

```suggestion
# /usr/bin/env python3
"""
File: filename.py
-----------------
"""
```

Reply:
After this, I'm good to merge (@cooper-mj)